src/http-auth-interceptor.js

```diff
@@ -10,7 +10,18 @@
 
   angular.module('http-auth-interceptor', ['http-auth-interceptor-buffer'])
 
-  .factory('authService', ['$rootScope','httpBuffer', function($rootScope, httpBuffer) {
+  	/**
+	 * Stores the config received upon login confirmation for the next requests
+	 */
+	.factory('authConfigService', function() {
+		return {
+			config : {
+				headers: {}
+			}
+		};
+	})
+  
+  .factory('authService', ['$rootScope','httpBuffer', 'authConfigService', function($rootScope, httpBuffer, authConfigService) {
```

Comment:
Some lines (like this one) are so long I can't even see what's there in GitHub's default code review page.